fineract-provider/src/main/java/org/apache/fineract/infrastructure/dataqueries/api/DataTableApiConstant.java

```diff
@@ -34,4 +34,25 @@ private DataTableApiConstant() {
     public static final String localParamName = "locale";
     public static final String DATATABLE_RESOURCE_NAME = "dataTables";
 
+    public static final String createdAtFieldName = "created_at";
```

Comment:
This naming is not comforming the Java styling standards.
`
The names of variables declared class constants and of ANSI constants should be all uppercase with words separated by underscores ("_"). (ANSI constants should be avoided, for ease of debugging.)` - https://www.oracle.com/java/technologies/javase/codeconventions-namingconventions.html